mica-webapp/src/main/resources/_templates/libs/navbar-menus-right.ftl

```diff
@@ -28,7 +28,7 @@
   <#if user??>
     <#if isAdministrator || isReviewer || isEditor || hasPermissionOnAnyDraftDocument>
       <li class="nav-item">
-        <a href="${contextPath}/administration" class="nav-link">
+        <a href="${contextPath}/<#if hasPermissionOnAnyDraftDocument>admin<#else>administration</#if>" class="nav-link">
```

Comment:
why not putting this condition at line 29 and updating the administration page accordingly?

Reply:
You still need to update the administration page: no need to expose the link to agate for someone who has not an advanced role

![image](https://github.com/obiba/mica2/assets/2581149/e9d66bb0-a3ff-4642-8b1e-409c7ed3b23d)

Reply:
I fixed it.